virt/virt-2-4-release-notes.adoc

```diff
@@ -18,6 +18,9 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 
 //CNV-5371 - Import existing virtual machines from Red Hat Virtualization to OpenShift Virtualization.
 
+//CNV-5439 - TLS certificates
+TLS certificates are automatically deleted and replaced for KubeVirt, Containerized Data Importer controller (CDI), and MAC pool. Automatic TLS certificate rotation does not disrupt any operations. All operations including migrations, image uploads, and VNC and console connections continue to function without any disruption.
```

Comment:
Looks good, but I'm unsure of the official name of "MAC pool". @phoracek should confirm.

Reply:
I think we use this in other places of OCP doc too.

The thing is, we only advertise the feature of MAC pooling, not the component providing it. But then, it feels weird to say that this feature rotates its TLS.

I'm fine with both "MAC pool" and "KubeMacPool" in this context. Neither of them is a perfect fit.

Reply:
We could probably still make it more crisp and just say that `Openshift virtualization`, `CNV` or `Openshift Virtualization` rotates certificates now automatically without extra listing the components.